packages/service-library/Makefile

```diff
@@ -4,41 +4,54 @@
 include ../../scripts/common.Makefile
 
 help:	## overwrites and calls common help function
-	$(MAKE) -f ../../scripts/common.Makefile $@
+	$(MAKE) --no-print-directory --file ../../scripts/common.Makefile $@
+	@echo
 	@echo '🚨 BEWARE there are issues with brakets in names `make test[aiohttp]` will raise errors:'
 	@echo '✅` `make "test[aiohttp]"` works as intended'
 	@echo '👓 Please note: when INSTALLING, DEVELOPING and TESTING '
+	@echo
 	@echo 'servicelib support for aiohttp extras is not included:'
-	@echo '- `make install-dev`: without aiohttp extras'
-	@echo '- `make "install-dev[aiohttp]"`: with aiohttp extras'
-    
-.PHONY: install-dev install-dev[aiohttp] install-ci install-ci[aiohttp]
-install-dev install-dev[aiohttp] install-ci install-ci[aiohttp]: _check_venv_active ## install app in development/production or CI mode
+	@echo '- `make install-dev`: common features w/o any extras'
+	@echo '- `make "install-dev[aiohttp]"`: w/ aiohttp extras'
+	@echo '- `make "install-dev[fastapi]"`: w/ fastapi extras'
+	@echo '- `make "install-dev[all]"`: w/ all extras'
```

Comment:
by all extras you mean both aiohttp and fastapi I guess?

Reply:
yes, in the [setuptools jargon](https://hanxiao.io/2019/11/07/A-Better-Practice-for-Managing-extras-require-Dependencies-in-Python/), those are called *extras-require*